mmv1/third_party/terraform/.teamcity/components/inputs/services_beta.kt

```diff
@@ -3,7 +3,8 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-// This file is maintained in the GoogleCloudPlatform/magic-modules repository and copied into the downstream provider repositories. Any changes to this file in the downstream will be overwritten.
```

Comment:
Could you remove the formatting changes for this file & just make your change? We don't have a standard formatter for kotlin files, so I don't want to introduce whitespace etc. changes that might just get reverted later.